flow360/component/flow360_params/flow360_params.py

```diff
@@ -194,13 +196,13 @@ class SupersonicInflow(Boundary):
 
     Example
     -------
-    >>> supersonicInflow = SupersonifInflow(density=1.0, velocity=[2.5, 0.1, 0], pressure=0.7) # doctest: +SKIP
+    >>> supersonicInflow = SupersonifInflow(totalTemperatureRatio=2.1, totalPressureRatio=3.0, staticPressureRatio=1.2)
     """
 
     type = pd.Field("SupersonicInflow", const=True)
-    density: PositiveFloat = pd.Field(alias="Density")
-    velocity: Tuple[float, float, float] = pd.Field(alias="Velocity")
-    pressure: PositiveFloat = pd.Field(alias="Pressure")
+    totalTemperatureRatio: PositiveFloat
+    totalPressureRatio: PositiveFloat
+    staticPressureRatio: PositiveFloat
```

Comment:
we use snake_case for parameter names and camelCase for export therefore we use 
```python
snake_case: Type = pd.Field(alias="camelCase")
```

```suggestion
    total_temperature_ratio: PositiveFloat = pd.Field(alias="totalTemperatureRatio")
    total_pressure_ratio: PositiveFloat = pd.Field(alias="totalPressureRatio")
    static_pressure_ratio: PositiveFloat = pd.Field(alias="staticPressureRatio")
```